cnf-certification-test/identifiers/identifiers.go

```diff
@@ -218,8 +229,10 @@ The label value is not important, only its presence.`,
 		`Checks that pods do not place persistent volumes on local storage.`,
 		StorageRequiredPods,
 		InformativeResult,
+		NoDocumentedProcess,
```

Comment:
This entry did not previously have an exception string (if you look at CATALOG.md on main all of the values from here on are off by one). If there is actually an exception process just let me know and I'll add it.

Reply:
Good catch!